frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -216,4 +216,18 @@ export const billingLogic = kea<billingLogicType>([
             }
         },
     })),
+    urlToAction(({ actions }) => ({
+        '/ingestion/billing': (_, { reason }) => {
+            if (reason === 'cancelled') {
+                eventUsageLogic.actions.reportIngestionBillingCancelled()
+            }
+        },
+        '/organization/billing/subscribed': (_, { referer }) => {
+            let successRedirect = urls.projectHomepage()
```

Comment:
as a nit

I think we should call an action: `actions.referer` here and then have a `billingSuccessRedirect` reducer which defaults to `urls.projectHomepage` and sets `urls.events` if referer is ingestion. Instead of having `setBillingSuccessRedirect`

Makes the reason and the effect clearer

Reply:
Done, makes total sense, much cleaner!